README.md

```diff
@@ -4,18 +4,19 @@ Basic TinaCMS starter based on [Next.js](https://nextjs.org) and [TinaCMS](https
 
 ![Starter Homepage Preview](public/uploads/tina-cloud-starter-preview.png)
 
-## Project scope:
+## Video Tour
 
-- Run this project locally using local content within this repository.
-- Connect to Tina Cloud to benefit from its GraphQL Content API.
-- Deploy the site to visually edit your site.
-- Invite collaborators.
+Get started in 10 minutes
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Y-fG7qzoHKw" title="Getting started with Tina Cloud in 10 minutes" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Looks like the video is not showing up.
![Screen Shot 2021-06-02 at 8 51 27 AM](https://user-images.githubusercontent.com/43075109/120475544-df53b500-c37f-11eb-82b9-bb0489695bc4.png)

Maybe we could do
```suggestion
Prefer a video explanation? [Watch a getting started video](https://youtu.be/Y-fG7qzoHKw)
```